provider/pact-jvm-provider/src/main/kotlin/au/com/dius/pact/provider/reporters/AnsiConsoleReporter.kt

```diff
@@ -45,7 +45,7 @@ class AnsiConsoleReporter(
 
   override fun finaliseReport() { }
 
-  override fun reportVerificationForConsumer(consumer: IConsumerInfo, provider: IProviderInfo) {
+  override fun reportVerificationForConsumer(consumer: IConsumerInfo, provider: IProviderInfo, tag: String?) {
```

Comment:
Should use the tag here too, in case anyone is relying on the console output.

Reply:
fixed